client/web/src/enterprise/batches/create/CreateOrEditBatchChangePage.tsx

```diff
@@ -133,9 +151,77 @@ interface CreatePageProps extends SettingsCascadeProps<Settings> {
      * default to the user's own namespace.
      */
     namespaceID?: Scalars['ID']
+    /** Display configuration in read-only mode */
+    isReadOnly?: boolean
+    /** Name of batch change when in read-only mode */
+    batchChangeName?: string
+}
+
+const CreatePage: React.FunctionComponent<CreatePageProps> = props => {
+    const isNewBatchChange = props.batchChangeName === undefined && !props.isReadOnly
+
+    return (
+        <div className="w-100 p-4">
+            <PageTitle title="Create new batch change" />
+            <PageHeader
+                path={[{ icon: BatchChangesIcon, to: '.' }, { text: 'Create batch change' }]}
+                className="flex-1 pb-2"
+                description="Run custom code over hundreds of repositories and manage the resulting changesets."
+                annotation={<FeedbackBadge status="experimental" feedback={{ mailto: 'support@sourcegraph.com' }} />}
+            />
+            <Tabs>
+                <TabList>
+                    <Tab className="text-content py-2 px-3">1. Configuration</Tab>
+                    <Tab className="text-content py-2 px-3" disabled={isNewBatchChange}>
+                        2. Batch Specification
```

Comment:
I'm sorry, this is also wrong in the designs - it's a word I can't get out of Robs vocabulary 😆 the correct term for the file is batch spec, as we name it in the docs :) 

```suggestion
                        2. Batch spec
```